src/main/java/mods/railcraft/data/advancements/RailcraftAdvancementProvider.java

```diff
@@ -0,0 +1,21 @@
+package mods.railcraft.data.advancements;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import net.minecraft.core.HolderLookup;
+import net.minecraft.data.PackOutput;
+import net.minecraftforge.common.data.ExistingFileHelper;
+import net.minecraftforge.common.data.ForgeAdvancementProvider;
+
+public class RailcraftAdvancementProvider extends ForgeAdvancementProvider {
```

Comment:
Remove when this [PR](https://github.com/MinecraftForge/MinecraftForge/pull/9212) will be merged